packages/suite/src/hooks/wallet/useCoinmarketBuyOffers.ts

```diff
@@ -127,7 +127,7 @@ export const useOffers = (props: Props) => {
         const quote = { ...selectedQuote, receiveAddress: address };
         const response = await invityAPI.doBuyTrade({
             trade: quote,
-            returnUrl: createTxLink(selectedQuote, account),
+            returnUrl: await createTxLink(selectedQuote, account),
```

Comment:
declare `returnUrl` as a const before the invityAPI call.

async in async looks suspicious